Sources/SwiftParserDiagnostics/ParseDiagnosticsGenerator.swift

```diff
@@ -913,6 +913,28 @@ public class ParseDiagnosticsGenerator: SyntaxAnyVisitor {
     return .visitChildren
   }
 
+  public override func visit(_ node: OptionalBindingConditionSyntax) -> SyntaxVisitorContinueKind {
+    if shouldSkip(node) {
+      return .skipChildren
+    }
+
+    if node.bindingKeyword.presence == .missing {
+      addDiagnostic(
+        node.bindingKeyword,
+        ExpectedBindingKeyword(token: node.bindingKeyword),
+        fixIts: [
+          FixIt(
+            message: InsertFixIt(tokenToBeInserted: node.bindingKeyword),
+            changes: [.makePresent(node.bindingKeyword)]
+          )
+        ],
+        handledNodes: [node.bindingKeyword.id]
+      )
+    }
```

Comment:
Out of curiosity: What’s the error message if we don’t have this custom one? My expectation is that it would be equally good and we could save ourselves some code. But I might be wrong.

Reply:
They are more or less the same.

Reply:
Removed this